test/Microsoft.AspNetCore.Razor.Test/Utils/DisposableActionTest.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 
-using Microsoft.AspNetCore.Razor.Utils;
+using Microsoft.AspNetCore.Razor.Parser.Internal;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Razor.Test.Utils
```

Comment:
Should change the namespace of the test to match the tested class. No need for the added `using`.